automation/Extension/firefox/src/privileged/sockets/api.js

```diff
@@ -79,20 +79,20 @@ this.sockets = class extends ExtensionAPI {
           });
         },
 
-        onDataReceived: new ExtensionCommon.EventManager(
-          context,
-          "sockets.onDataReceived",
-          (fire) => {
+        onDataReceived: new ExtensionCommon.EventManager({
+          context: context,
+          name: "sockets.onDataReceived",
+          register: (fire) => {
             let listener = (id, data) => {
               fire.async(id, data);
             };
-
+            console.log(gManager.onDataReceivedListeners)
```

Comment:
hey, just wanted to point out this seemingly leftover console.log, but I think it's ok to leave it in.

Reply:
Good catch! I missed that. It's easy enough to clean up though.

@vringar mind to make a PR that removes this?